python/distance_calculator.py

```diff
@@ -0,0 +1,112 @@
+import mgp
+import math
+import itertools
+
+KM_MULTIPLIER = 0.001
+LATITUDE = 'lat'
+LONGITUDE = 'lng'
+valid_metrics = ['m', 'km']
```

Comment:
Just like you did correctly for `KM_MULTI...`, `LATITUDE` and `LONGITUDE` it would be great to have this as `VALID_METRICS` because it is a constant.

Reply:
It should be fixed in both files using the constant.